### ISSUE TITLE
Extensions not being seen as installed on Preview v0.126.0

### ISSUE THREAD

**Issue by fuzenco** [zed-industries/zed#9028]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

After today’s Preview release (v0.126.0), all the extensions I installed last week are not shown as installed. So the Installed tab in the UI shows no resulting extensions, but simply an empty page. And the Not Installed tab returns all extensions; even the many I already have installed.

### Environment

Zed: v0.126.0 (Zed Preview)
OS: macOS 14.3.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.

[Zed.log](https://github.com/zed-industries/zed/files/14528492/Zed.log)

**Comment by maxdeviant** [comment]
This is a known issue.

Running the `zed: reload extensions` command from the command palette should bring them back.

**Comment by fuzenco** [comment]
And by extensions, I mostly mean Themes. I will need to revisit my themes to make sure they weren’t lost when I updated.

**Comment by fuzenco** [comment]
One last comment: Quite honestly, I know I installed at least 15 themes. None of which show up in the Installed list. And looking in ~/.config/zed/themes, I only see the two themes I manually placed there before I learned of the Extensions feature (being able to install themes through a GUI).

So perhaps the themes showing up in my Theme Selector (via the command palette) are default themes that ship with Zed. All other themes I installed via the GUI are perhaps not being recognized?
